targets/PythonSdk/source/source/PlayFabManualTest.py

```diff
@@ -0,0 +1,57 @@
+# idea here is to include PlayFabSettings, set them,
+# include API.py (or PlayFabSErverAPI.py) and see if you can call a function out of it
+import PlayFabSettings
+import PlayFabEntityAPI
+import PlayFabServerAPI
+import PlayFabClientAPI
+import PlayFabManualTestSettings
+
+PlayFabSettings.TitleId = PlayFabManualTestSettings.TitleId
+PlayFabSettings.SdkVersionString = PlayFabManualTestSettings.SdkVersionString
```

Comment:
PlayFabSettings.SdkVersionString should definitely not be altered.
It should be correctly generated as a constant in that file, and we never suggest that it should be modified.